cypress/integration/stackFeatures.js

```diff
@@ -0,0 +1,35 @@
+import { randHash } from '../utils'
+const randUser = randHash()
+
+describe('Stack', function () {
+    const board = 'TestBoard'
+    const password = 'pass123'
+    const stack = 'List 1'
+
+    before(function () {
+		cy.nextcloudCreateUser(randUser, password)
+        cy.deckCreateBoard({ user: randUser, password }, board)
+	})
+
+	beforeEach(function() {
+        cy.logout()
+		cy.login(randUser, password)
+	})
+
+    it('Can create a stack', function () {
+        
+        cy.get('.app-navigation button.app-navigation-toggle').click()
+        cy.get('#app-navigation-vue .app-navigation__list .app-navigation-entry')
+            .eq(3)
+            .find('a.app-navigation-entry-link')
+            .first().click({force: true})
+
+        cy.get('#stack-add button').first().click()
+        cy.get('#stack-add form input#new-stack-input-main')
+            .type(stack)
+        cy.get('#stack-add form input[type=submit]')
+            .first().click()
+
+        cy.get('.board .stack').eq(0).contains(stack).should('be.visible')
```

Comment:
Can we also check for the stack title in here?

Reply:
Sorry but I didn't really get your point. "contains with should" assertion checks the tile in both cases: https://docs.cypress.io/api/commands/contains.
card's content is 'Card 1' and stack's value is 'Stack 1'

Reply:
Same here, sorry for the noise